signature/fulcio_cert.go

```diff
@@ -185,21 +185,6 @@ func (f *fulcioTrustRoot) verifyFulcioCertificateAtTime(relevantTime time.Time,
 	return untrustedCertificate.PublicKey, nil
 }
 
-func parseLeafCertFromPEM(untrustedCertificateBytes []byte) (*x509.Certificate, error) {
```

Comment:
Move also `TestParseLeafCertsFromPEM` (and `s/Certs/Cert/`) then, please.

Reply:
done